foreman/data_refinery_foreman/surveyor/sra.py

```diff
@@ -467,10 +475,7 @@ def _generate_experiment_and_samples(self, run_accession: str, study_accession:
                 sample_object.manufacturer = "UNKNOWN"
 
             # Directly apply the harmonized values
```

Comment:
I think we don't need this comment any more since it's essentially in the function name/docstring.